js/index.js

```diff
@@ -138,6 +139,12 @@ Raven.context(() => {
       }
     ],
     enableHotkeys: true,
+    __extraWindows: {
+      MILKDROP_WINDOW: {
+        title: "Milkdrop 2",
+        Component: MilkdropWindow
+      }
+    },
```

Comment:
I've merged https://github.com/captbaritone/webamp/commit/569a6819f0a9740f18e80d12ea5a006c250fca8c so this can just become: 

```JavaScript
__extraWindows: [{ title: "Milkdrop 2", Component: MilkdropWindow}]
```

Reply:
awesome!